src/middleware.ts

```
@@ -13,22 +12,13 @@ const corsOptions = {
-    return !!session?.user;
-  } catch (error) {
-    console.error('Session validation error:', error);
-    return false;
-  }
+
+function hasAuthCookies(request: NextRequest): boolean {
+  // Check for better-auth session cookies
+  const sessionCookie = request.cookies.get('better-auth.session_token');
+  const csrfCookie = request.cookies.get('better-auth.csrf_token');
+  
+  return !!(sessionCookie && sessionCookie.value);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)